test.js

```diff
@@ -360,6 +360,12 @@ test('encryption - corrupt file', t => {
 	t.is(after.get('foo'), undefined);
 });
 
+test('decription migration to IV', t => {
+	// The encrypted_config.json contain '{'unicorn', '🦄'}' encrypted with conf@4.1.0 and password 'abcd1234'
```

Comment:
It cannot contain `{'unicorn', '🦄'}` as it's not valid JSON. Please include *exactly* what it contains.

Reply:
Sorry this is a typo.. I will say `{'unicorn': '🦄'}` which is created with `config.set('unicorn', '🦄')`.